### PR TITLE
Add java, js and ruby GHA workflows

### PR DESCRIPTION
**Pull request by aurelien-reeves**
# Description

Add workflows to the CI

### CODE REVIEW

**Comment by aurelien-reeves** [comment]
@aslakhellesoy FYI the acceptance tests still need to be migrated

**Comment by aslakhellesoy** [comment]
Yes, let’s use the same approach we use in tag-expressions and cucumber-expressions where we have native tests that read the acceptance test data files.

**Comment by aurelien-reeves** [comment]
Ruby and JavaScript acceptance tests migrated from make to rspec/mocha.

Need help with Java 😭

File: javascript/src/createMeta.ts [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-import { parse as parseUrl, format as formatUrl } from 'url'
 import * as messages from '@cucumber/messages'
+import os from 'os'
+import { format as formatUrl, parse as parseUrl } from 'url'
+
 import defaultCiDict from './ciDict.json'
 import evaluateVariableExpression from './evaluateVariableExpression'
 import { CiDict, CiSystem, Env } from './types'
 
+interface gitHash {
```
Comment (aslakhellesoy) [inline javascript/src/createMeta.ts:9]: Use a capital letter for types. Prefer `type` over `interface` when there are no methods. `gitHash` sounds strange to me. How about `GitInfo`?

**Comment by aslakhellesoy** [comment]
@aurelien-reeves look at how it's done in https://github.com/cucumber/cucumber-expressions/blob/main/java/src/test/java/io/cucumber/cucumberexpressions/CucumberExpressionParserTest.java - let me know if you want to pair.

File: javascript/src/createMeta.ts (hunk 1 above)
Comment (olleolleolle) [reply javascript/src/createMeta.ts]: (Aside: Perhaps a [Git "tree-ish"](https://git-scm.com/docs/gitglossary#def_tree-ish)? But, the remote is not a part of that, though.)

Comment (aurelien-reeves) [reply javascript/src/createMeta.ts]: Indeed it looks a little bit like a tree-ish

GitInfo seems more generic, and maybe more common in a context like the one in CreateMeta.

(I was not convince with gitHash myself :p)

**Comment by aurelien-reeves** [comment]
@aslakhellesoy I have been able to make some progress with the java acceptance testing

But there is still some work to do with those, I am far from having finished those.

**Pull request merged**